packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -157,6 +157,35 @@ function FlowsheetDataGrid(props) {
     isResizable: false,
   })), [columns]);
 
+  const tableBodyColumns = useMemo(() => {
```

Comment:
Why do we need both the flowsheetColumns and tableBodyColumns objects?  We should be able to combine the initialization logic and use a single object to simplify the code.